src/pages/Appointments/AppointmentsPage.tsx

```diff
@@ -681,6 +681,7 @@ function AppointmentColumn(props: {
         user: props.practitioner ?? undefined,
         date_after: props.date_from,
         date_before: props.date_to,
+        ordering: "-token_slot__start_datetime",
```

Comment:
It should be reverse

```suggestion
        ordering: "token_slot__start_datetime",
```